docs/models.yml

```diff
@@ -1629,10 +1569,7 @@ option:
         - collection: topic
```

Comment:
can this be
```
to:
  - motion/option_ids
  - ...
```

This would simplyfy the schema a lot, when only accepting `to: <string>` and `to: <string>[]`.

Reply:
Sure, for the backend this is irrelevant. Just for clarification: For fields with multiple collections which have the same name, do you still want to keep the schema:
```
to:
  collection:
    - a
   ...
  field: ...
```
or do you want to repeat the field name in each definition here?

Reply:
> or do you want to repeat the field name in each definition here?

Would be simpler imo

Reply:
I kept it in to avoid repetition. Does not work as soon as one field falls out of line of course, e.g. for the projector. Anyway, should be ready now